pkg/config/deployment.go

```diff
@@ -583,16 +583,16 @@ func (dc *DeploymentChain) Validate() error {
 	return nil
 }
 
-// DeploymentChainNode provides filters used to find the right applications to trigger
+// ChainApplicationMatcher provides filters used to find the right applications to trigger
 // as a part of the deployment chain.
-type DeploymentChainNode struct {
+type ChainApplicationMatcher struct {
 	AppName   string            `json:"name"`
```

Comment:
"Name" is enough.

Reply:
Nice point 👍  addressed by https://github.com/pipe-cd/pipe/pull/2820/commits/91d35eee7a6fabe3dd269f62865f435228dc3162